scripts/dev/generate_epJSON_schema/modify_schema.py

```diff
@@ -170,6 +170,7 @@ def isInt(s):
     'Foundation:Kiva': 'blocks',
     'SurfaceProperty:ExposedFoundationPerimeter': 'surfaces',
     'SurfaceProperty:SurroundingSurfaces': 'surfaces',
+	'SurfaceProperty:GroundSurfaces': 'ground_surfaces',
```

Comment:
Another tab...I'd check your editor for why it is putting in tabs instead of spaces.  I'll fix it here since I already have the IDD tab fixes done locally.